### PR TITLE
GitHub salt module

### PR DESCRIPTION
**Pull request by yashsarma**
Salt module to manage github users in your organization.

### CODE REVIEW

File: salt/modules/github.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# -*- coding: utf-8 -*-
+'''
+Module for interop with the Github v3 API
+
+.. versionadded:: 2015.2.0
```
Comment (jfindlay) [inline salt/modules/github.py]: `.. versionadded:: Boron`

File: salt/states/github.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+'''
+Github User State Module
+
+.. versionadded:: 2015.2.0
```
Comment (jfindlay) [inline salt/states/github.py]: `.. versionadded:: Boron`

**Comment by cachedout** [comment]
Awesome!! Thanks!

**Comment by mostafahussein** [comment]
@yashlyft  why this is a module and not a runner and what is the point of managing github users through salt? Could you explain please  the point of salt integration ?

**Pull request merged**